mirascope/anthropic/calls.py

```diff
@@ -62,26 +68,19 @@ def call(
             A `AnthropicCallResponse` instance.
         """
         messages, kwargs, tool_types = self._setup_anthropic_kwargs(kwargs)
-        client = Anthropic(api_key=self.api_key, base_url=self.base_url)
-        if self.call_params.wrapper is not None:
-            client = self.call_params.wrapper(client)
-
+        client = get_wrapped_client(
+            Anthropic(api_key=self.api_key, base_url=self.base_url), self
+        )
         create = client.messages.create
         if tool_types:
             create = client.beta.tools.messages.create  # type: ignore
-        if self.call_params.weave is not None:
-            create = self.call_params.weave(create)  # pragma: no cover
-        if self.call_params.logfire:
-            create = self.call_params.logfire(
+        if len(self.configuration.llm_ops) > 0:
```

Comment:
since this is of type `list` and defaults to `[]` you can just check `if self.configuration.llm_ops`

Update this here and elsewhere please.

Reply:
done